factory/reader.js

```diff
@@ -22,8 +22,43 @@ function parseContent(html) {
   var $ = cheerio.load(html),
       articleMarkup = $('.post-outer div.post.hentry').html(),
       content = getTextFromHtml(articleMarkup);
-
-  return content;
+      
+      //@dfasoro added this.
+      var comments = [];
+      var commentsNode = $('#comments');
+      comments.push($('H4', commentsNode).first().text());
+      
+      var authors = [], body = [], time = [];
+      
+      try {
+          $('#comments-block DT.comment-author', commentsNode).each(function () {
+              var author_line = $(this).text().trim();
+              author_line = author_line.substr(0, author_line.length - 7).trim();
+              authors.push(author_line);
+          });
+          $('#comments-block DD.comment-body', commentsNode).each(function () {
+              var body_line = getTextFromHtml($(this).html()).trim();
+              body.push(body_line);
+          });
+          $('#comments-block DD.comment-footer SPAN.comment-timestamp > A', commentsNode).each(function () {
+              var time_line = $(this).text().trim();
+              time.push(time_line);
+          });
+          //console.log(authors);
+          //console.log(body);
+          //console.log(time);
+      }
+      catch (pex) {
+          //any yawa that gas should not make the original post not to show up right.
+      }
+      
+      authors.forEach(function (value, index, array) {
+          
+          var text = `${index + 1}) ${authors[index]} @ ${time[index]} said:\n${body[index]}\n`;
+          comments.push(text);
+      });
+      
+  return content + "\n\n" + comments.join("\n");
```

Comment:
Please use spaces instead of tabs for the indentation. Two spaces to be specific for uniformity and clean code appearance.